src/lib/viewers/iframe/IFrameLoader.js

```diff
@@ -22,6 +23,20 @@ class IFrameLoader extends AssetLoader {
         super();
         this.viewers = VIEWERS;
     }
+
+    /**
+     * @inheritdoc
+     */
+    determineViewer(file, disabledViewers = [], viewerOptions = {}) {
+        const isDicomFile = file.name === 'Dicom.boxdicom' || file.extension === 'boxdicom';
+        const openWithAmbraEnabled = getProp(viewerOptions, 'IFrame.openWithAmbra');
```

Comment:
Instead of openWithAmbra, maybe the option should be `disableDicom` focused because preview SDK knows nothing about an Ambra integration